lib/lotus/utils/escape.rb

```diff
@@ -0,0 +1,487 @@
+module Lotus
+  module Utils
+    # HTML escape utilities
+    #
+    # Based on OWASP research and OWASP ESAPI code
+    #
+    # @since x.x.x
+    #
+    # @see https://www.owasp.org
+    # @see https://www.owasp.org/index.php/Cross-site_Scripting_%28XSS%29
+    # @see https://www.owasp.org/index.php/XSS_(Cross_Site_Scripting)_Prevention_Cheat_Sheet
+    # @see https://www.owasp.org/index.php/ESAPI
+    # @see https://github.com/ESAPI/esapi-java-legacy
+    module Escape
+      # Hex base for base 10 integer conversion
+      #
+      # @since x.x.x
+      # @api private
+      #
+      # @see http://www.ruby-doc.org/Fixnum.html#method-i-to_s
```

Comment:
Page not found, the message in the web:

`We're sorry, but that page cannot be found.`